core/arch/arm/plat-imx/config/imx6q.h

```diff
@@ -0,0 +1,267 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2015 Freescale Semiconductor, Inc.
+ * Copyright (c) 2016, Wind River Systems.
+ * All rights reserved.
+ * Copyright 2017-2018 NXP
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef PLATFORM_CONFIG_IMX6Q_H
+#define PLATFORM_CONFIG_IMX6Q_H
+
+/* Board specific console UART */
+#if defined(PLATFORM_FLAVOR_mx6qsabrelite)
+#define CONSOLE_UART_BASE		UART2_BASE
+#endif
+#if defined(PLATFORM_FLAVOR_mx6qpsabreauto) || \
+	defined(PLATFORM_FLAVOR_mx6qsabreauto) || \
+	defined(PLATFORM_FLAVOR_mx6dlsabreauto)
+#define CONSOLE_UART_BASE		UART4_BASE
+#endif
+#ifndef CONSOLE_UART_BASE
+#define CONSOLE_UART_BASE		UART1_BASE
+#endif
+
+/* Board specific RAM size */
+#if defined(PLATFORM_FLAVOR_mx6qsabrelite) || \
+	defined(PLATFORM_FLAVOR_mx6qsabresd) || \
+	defined(PLATFORM_FLAVOR_mx6qsabrelite) || \
+	defined(PLATFORM_FLAVOR_mx6dlsabresd) || \
+	defined(PLATFORM_FLAVOR_mx6qpsabresd)
+#define DRAM0_SIZE			0x40000000
+#endif
+
+#if defined(PLATFORM_FLAVOR_mx6qpsabreauto) || \
+	defined(PLATFORM_FLAVOR_mx6qsabreauto) || \
+	defined(PLATFORM_FLAVOR_mx6dlsabreauto)
+#define DRAM0_SIZE			0x80000000
+#endif
+
+/* Core number depends of SoC version. */
+#if defined(CFG_MX6QP) || defined(CFG_MX6Q)
+#define CFG_TEE_CORE_NB_CORE		4
+#endif
+#if defined(CFG_MX6D) || defined(CFG_MX6DL)
+#define CFG_TEE_CORE_NB_CORE		2
+#endif
+#if defined(CFG_MX6S)
+#define CFG_TEE_CORE_NB_CORE		1
+#endif
+
+/* Common RAM and cache controller configuration */
+#define CFG_TEE_RAM_VA_SIZE		(1024 * 1024)
```

Comment:
it would be nice to have this near the other `CFG_TEE_RAM_xxx`

Reply:
sure

Reply:
please do it.